v-next/hardhat/src/internal/cli/help/utils.ts

```diff
@@ -9,7 +9,7 @@ import { camelToKebabCase } from "@nomicfoundation/hardhat-utils/string";
 
 export const GLOBAL_NAME_PADDING = 6;
 
-interface ArgumentDescriptor {
+export interface ArgumentDescriptor {
```

Comment:
We always careful about the types we export. We make a distinction between those that are available to consumers of the package (which are available at the top level) and those under `./internal`. Those under `./internal` are not exposed and we can manipulate and move without concern for the external API.
In this case I would be tempted to move this interface as its now shared among files to `./src/internal/cli/utils.ts` - a new file, and update the imports.
But I think it is borderline, so I will leave it to your judgement.